src/System.ServiceModel.Primitives/ref/System.ServiceModel.Primitives.cs

```diff
@@ -1380,6 +1392,7 @@ public partial class MessagePropertyDescriptionCollection : System.Collections.O
     public partial class OperationDescription
     {
         public OperationDescription(string name, System.ServiceModel.Description.ContractDescription declaringContract) { }
+        public System.Collections.Generic.KeyedByTypeCollection<System.ServiceModel.Description.IOperationBehavior> Behaviors { get { return default(System.Collections.Generic.KeyedByTypeCollection<System.ServiceModel.Description.IOperationBehavior>); } }
```

Comment:
This needs the following attribute applied to prevent intellisense from showing it. This is to match the full framework.
```csharp
[EditorBrowsable(EditorBrowsableState.Never)]
```